app/util/job.ts

```diff
@@ -74,11 +74,10 @@ export async function completeJob(
     job.updateStatus(finalStatus, message);
     await job.save(tx);
     if (failed) {
-      const numUpdated = await updateWorkItemStatusesByJobId(
+      await updateWorkItemStatusesByJobId(
         tx, job.jobID, [WorkItemStatus.READY, WorkItemStatus.RUNNING, WorkItemStatus.QUEUED], WorkItemStatus.CANCELED,
       );
-      const itemMeta: WorkItemMeta = { workItemStatus: WorkItemStatus.CANCELED, workItemAmount: numUpdated, workItemEvent: 'statusUpdate' };
-      logger.info(`Updated work items to ${WorkItemStatus.CANCELED} for completed job.`, itemMeta);
+      logger.info(`Updated work items to ${WorkItemStatus.CANCELED} for completed job.`);
```

Comment:
I would keep the number of work items updated in this log statement even if you don't want to keep the rest of the WorkItemMeta fields.